From 15ee68074ba84ca0d433a484d59b7ef9a23115d9 Mon Sep 17 00:00:00 2001
From: Bastian Blank <waldi@debian.org>
Date: Tue, 30 Jun 2015 11:46:09 +0200
Subject: implicit-pointer

---
 tools/lvm.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/tools/lvm.c b/tools/lvm.c
index 4bfa6f5c25..65bc88df4b 100644
--- a/tools/lvm.c
+++ b/tools/lvm.c
@@ -26,10 +26,6 @@ int main(int argc, char **argv)
 
 #  include <readline/readline.h>
 #  include <readline/history.h>
-#  ifndef HAVE_RL_COMPLETION_MATCHES
-#    define rl_completion_matches(a, b) completion_matches((char *)a, b)
-#    define rl_completion_func_t CPPFunction
-#  endif
 
 static struct cmdline_context *_cmdline;
 
