Index: lvm2/scripts/blkdeactivate.sh.in
===================================================================
--- lvm2.orig/scripts/blkdeactivate.sh.in	2021-02-22 20:47:47.840061762 +0000
+++ lvm2/scripts/blkdeactivate.sh.in	2021-02-22 20:48:45.153366726 +0000
@@ -60,7 +60,7 @@
 LSBLK="/bin/lsblk -r --noheadings -o TYPE,KNAME,NAME,MOUNTPOINT"
 LSBLK_VARS="local devtype local kname local name local mnt"
 LSBLK_READ="read -r devtype kname name mnt"
-SORT_MNT="/bin/sort -r -u -k 4"
+SORT_MNT="/usr/bin/sort -r -u -k 4"
 
 # Do not show tool errors by default (only done/skipping summary
 # message provided by this script) and no verbose mode by default.
